src/main/java/seedu/address/logic/commands/commandresult/CommandResult.java

```diff
@@ -10,7 +10,7 @@
 /**
  * Represents the result of a command execution.
  */
-public class CommandResult {
+public abstract class CommandResult {
```

Comment:
Making this an abstract class makes the code cleaner and creating new classes that inherits it is a better coding practise.
Thank you!